Documentation/devel/pod-lifecycle.md

```diff
@@ -51,7 +55,9 @@ After preparation completes, while still holding the exclusive lock (the lock is
 
 `rkt run` transitions directly from #Prepare to #Run by renaming `$var/prepare/$uuid` to `$var/run/$uuid`, entirely skipping the #Prepared phase.
```

Comment:
Is the switch from `/var/run/...` to the `$var/run/...` notation explained anywhere? What is `$var`?